app/views/admin/users/index/_controls.html.erb

```diff
@@ -15,6 +15,9 @@
             <span class="search-indicator absolute top-1 right-1 c-indicator c-indicator--info <%= params[:role].blank? ? "hidden" : "" %>"></span>
           </button>
         </div>
+        <div>
+          <%= render "admin/users/index/controls/export", f: f %>
+        </div>
```

Comment:
In all screen sizes, the layout position of this button is a bit different from the designs (the designs always place pagination & export button at the far right of the layout, with everything else aligned to the opposite site).

I'm not sure if that's something we want to address in this PR or in a follow up (I don't mind either way).

Reply:
ooops, I had a picture of the designs **in my head that I was referencing** and didn't notice how I've laid it out differently even when I did look at the figma 😅 I'll fix this now 😁

Reply:
Done ✅ 
<img width="676" alt="Screenshot 2022-04-14 at 15 32 39" src="https://user-images.githubusercontent.com/2786819/163402691-470ddf92-7b7a-4e08-8c60-418ff2b2c945.png">
<img width="1512" alt="Screenshot 2022-04-14 at 15 28 54" src="https://user-images.githubusercontent.com/2786819/163402745-b6403b95-8f68-45f8-9e01-1bf1c1d53c6c.png">

<img width="1512" alt="Screenshot 2022-04-14 at 15 28 54" src="https://user-images.githubusercontent.com/2786819/163402669-30604f9c-cca3-4040-afa1-985a0856e1bb.png">